airflow/utils/task_group.py

```diff
@@ -54,6 +54,8 @@ class TaskGroup(TaskMixin):
     :type ui_color: str
     :param ui_fgcolor: The label color of the TaskGroup node when displayed in the UI
     :type ui_fgcolor: str
+    :param from_decorator: was this taskgroup created by a taskgroup decorator (should only be set by Airflow)
+    :type from_decorator: bool
```

Comment:
```suggestion
    :param _from_decorator: was this taskgroup created by a taskgroup decorator (should only be set by Airflow)
    :type _from_decorator: bool
```


Or perhaps better might be:

```suggestion
    :param add_suffix_on_collision: If this task group name already exists, automatically add `__1` etc suffixes
    :type from_decorator: add_suffix_on_collision
```

(Which inverts the meaning of the bool)

The idea here is that "from_decorator" says nothing about what this parameter does, but add_suffix_on_collision should be immediately clear, especially at the caller.